### ISSUE TITLE
Unable to format a file that contains <!doctype html> 

### ISSUE THREAD

**Issue by wujekbogdan** [trivago/prettier-plugin-twig-melody#12]
The plugin fails to format a file if the file contains the `<!doctype html>` tag.

Here's the stack trace:

```
./node_modules/.bin/prettier --write ./views/base.twig
views/base.twig
<--- Last few GCs --->
ca[30127:0x1028c9000]     7706 ms: Mark-sweep 2023.2 (2027.8) -> 2023.1 (2027.8) MB, 1031.8 / 0.0 ms  (+ 14.9 ms in 3 steps since start of marking, biggest step 5.8 ms, walltime since start of marking 1051 ms) (average mu = 0.166, current mu = 0.119) alloca[30127:0x1028c9000]     8659 ms: Mark-sweep 2035.6 (2040.3) -> 2035.6 (2040.3) MB, 802.7 / 0.0 ms  (+ 16.6 ms in 5 steps since start of marking, biggest step 4.0 ms, walltime since start of marking 823 ms) (average mu = 0.154, current mu = 0.140) allocati

<--- JS stacktrace --->

==== JS stack trace =========================================

    0: ExitFrame [pc: 0x100930c99]
Security context: 0x0ee54bf808a1 <JSObject>
    1: next [0xee53f527461] [/Users/wujekbogdan/htdocs/my_site/node_modules/melody-parser/lib/index.js:~1359] [pc=0x38f30f0ecb0b](this=0x0ee57efc0a59 <Lexer map = 0xee500474281>)
    2: getAllTokens(aka getAllTokens) [0xee5f2765b71] [/Users/wujekbogdan/htdocs/my_site/node_modules/melody-parser/lib/index.js:~...

FATAL ERROR: Ineffective mark-compacts near heap limit Allocation failed - JavaScript heap out of memory

Writing Node.js report to file: report.20200101.165747.30127.0.001.json
Node.js report completed
 1: 0x10007e9b3 node::Abort() [/Users/wujekbogdan/.nodebrew/current/bin/node]
 2: 0x10007eb37 node::OnFatalError(char const*, char const*) [/Users/wujekbogdan/.nodebrew/current/bin/node]
 3: 0x100176337 v8::Utils::ReportOOMFailure(v8::internal::Isolate*, char const*, bool) [/Users/wujekbogdan/.nodebrew/current/bin/node]
 4: 0x1001762d3 v8::internal::V8::FatalProcessOutOfMemory(v8::internal::Isolate*, char const*, bool) [/Users/wujekbogdan/.nodebrew/current/bin/node]
 5: 0x1002fa485 v8::internal::Heap::FatalProcessOutOfMemory(char const*) [/Users/wujekbogdan/.nodebrew/current/bin/node]
 6: 0x1002fbb54 v8::internal::Heap::RecomputeLimits(v8::internal::GarbageCollector) [/Users/wujekbogdan/.nodebrew/current/bin/node]
 7: 0x1002f8a27 v8::internal::Heap::PerformGarbageCollection(v8::internal::GarbageCollector, v8::GCCallbackFlags) [/Users/wujekbogdan/.nodebrew/current/bin/node]
 8: 0x1002f6a0d v8::internal::Heap::CollectGarbage(v8::internal::AllocationSpace, v8::internal::GarbageCollectionReason, v8::GCCallbackFlags) [/Users/wujekbogdan/.nodebrew/current/bin/node]
 9: 0x100302124 v8::internal::Heap::AllocateRawWithLightRetry(int, v8::internal::AllocationType, v8::internal::AllocationAlignment) [/Users/wujekbogdan/.nodebrew/current/bin/node]
10: 0x10030219f v8::internal::Heap::AllocateRawWithRetryOrFail(int, v8::internal::AllocationType, v8::internal::AllocationAlignment) [/Users/wujekbogdan/.nodebrew/current/bin/node]
11: 0x1002ced97 v8::internal::Factory::NewFillerObject(int, bool, v8::internal::AllocationType) [/Users/wujekbogdan/.nodebrew/current/bin/node]
12: 0x1005f884d v8::internal::Runtime_AllocateInOldGeneration(int, unsigned long*, v8::internal::Isolate*) [/Users/wujekbogdan/.nodebrew/current/bin/node]
13: 0x100930c99 Builtins_CEntry_Return1_DontSaveFPRegs_ArgvOnStack_NoBuiltinExit [/Users/wujekbogdan/.nodebrew/current/bin/node]
14: 0x38f30f0ecb0b
[1]    30127 abort      ./node_modules/.bin/prettier --write ./views/base.twig
```

---
node: v12.13.1
os: macOS 10.15.2
prettier: ^1.19.1
prettier-plugin-twig-melody: ^0.1.0

**Comment by twbartel** [comment]
Hi @wujekbogdan, I just published version `0.2.0` which should fix this issue. Can you have a look?

**Comment by wujekbogdan** [comment]
It works fine now. Thank you!.
